client/setup.py

```diff
@@ -0,0 +1,74 @@
+#!usr/bin/python
+
+"Package installation setup"
+
+import os
+import subprocess
+from setuptools import setup, find_packages
+
+package_name = "pyroclient"
+with open(os.path.join(package_name, "version.py")) as version_file:
+    version = version_file.read().strip()
+sha = "Unknown"
+
+cwd = os.path.dirname(os.path.abspath(__file__))
+
+try:
+    sha = (
+        subprocess.check_output(["git", "rev-parse", "HEAD"], cwd=cwd)
+        .decode("ascii")
+        .strip()
+    )
+except Exception as e:
+    print(e)
+    pass
+
+if os.getenv("BUILD_VERSION"):
+    version = os.getenv("BUILD_VERSION")
+elif sha != "Unknown":
+    version += "+" + sha[:7]
+print("Building wheel {}-{}".format(package_name, version))
+
+
+with open("README.md") as f:
+    readme = f.read()
+
+with open("requirements.txt") as f:
+    requirements = f.read()
+
+
+setup(
+    name=package_name,
+    version=version,
+    author="Pyronear Contributors",
+    description="Client of the Pyronear API to help the fight against wildfires",
+    long_description=readme,
+    long_description_content_type="text/markdown",
+    url="https://github.com/pyronear/pyro-api",
+    download_url="https://github.com/pyronear/pyro-api/tags",
+    license="GPLv3",
+    classifiers=[
+        "Development Status :: 2 - Pre-Alpha",
+        "Intended Audience :: Developers",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+        "Natural Language :: English",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Scientific/Engineering :: Mathematics",
+        "Topic :: Scientific/Engineering :: Artificial Intelligence",
+        "Topic :: Software Development",
+        "Topic :: Software Development :: Libraries",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+    ],
+    keywords=["data science", "time series", "machine learning", "client", "api"],
```

Comment:
same here as we don't do any data science in the repo, here is a suggestion:
backend, api, client, wildfire